### PR TITLE
[MSTAGE-25] use Java 7 file handling in test

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

File: src/test/java/org/apache/maven/plugins/stage/RepositoryCopierTest.java [hunk 1]
```diff
@@ -103,28 +103,28 @@ private void testMavenArtifact( File repo, String artifact )
...
-
-        assertTrue( versions.contains( "2.0.3" ) );
-        
-        assertTrue( versions.contains( "2.0.4" ) );
-
-        assertTrue( versions.contains( "2.0.5" ) );
-
-        r.close();
+        File file = new File( basedir, RepositoryCopier.MAVEN_METADATA);
+        try ( Reader r = new InputStreamReader ( new FileInputStream( file ), StandardCharsets.UTF_8 ) )
```
Comment (michael-o) [inline src/test/java/org/apache/maven/plugins/stage/RepositoryCopierTest.java:107]: Why now `Files.newBufferedReader()`?

File: src/test/java/org/apache/maven/plugins/stage/RepositoryCopierTest.java [hunk 2]
```diff
@@ -103,28 +103,28 @@ private void testMavenArtifact( File repo, String artifact )
...
-        assertTrue( versions.contains( "2.0.2" ) );
-
-        assertTrue( versions.contains( "2.0.3" ) );
-        
-        assertTrue( versions.contains( "2.0.4" ) );
-
-        assertTrue( versions.contains( "2.0.5" ) );
-
-        r.close();
+        File file = new File( basedir, RepositoryCopier.MAVEN_METADATA);
```
Comment (michael-o) [inline src/test/java/org/apache/maven/plugins/stage/RepositoryCopierTest.java:106]: Space missing

**Review by michael-o — CHANGES_REQUESTED** [review]

File: src/test/java/org/apache/maven/plugins/stage/RepositoryCopierTest.java (hunk 1 above)
Comment (elharo) [reply src/test/java/org/apache/maven/plugins/stage/RepositoryCopierTest.java]: done

File: src/test/java/org/apache/maven/plugins/stage/RepositoryCopierTest.java (hunk 2 above)
Comment (elharo) [reply src/test/java/org/apache/maven/plugins/stage/RepositoryCopierTest.java]: fixed

**Review by michael-o — APPROVED** [review]

**Pull request merged**